daemon/cmd/daemon_main.go

```diff
@@ -1523,6 +1542,19 @@ func initKubeProxyReplacementOptions() {
 		}
 	}
 
+	if option.Config.EnableNodePort &&
+		option.Config.NodePortAcceleration != option.NodePortAccelerationNone {
+		if option.Config.XDPDevice != "undefined" &&
+			option.Config.XDPDevice != option.Config.Device {
+			log.Fatalf("Cannot set NodePort acceleration device: mismatch between Prefilter device %s and NodePort device %s",
```

Comment:
One extra fixup later, the check here is `option.Config.Device` but the message says "Prefilter" (which I would expect to be `option.Config.DevicePreFilter`).